trademapper/fragments/cookiedialog.html

```diff
@@ -1,14 +1,9 @@
-<div id='cookiedialog'>
+<div id='cookiedialog' role="alert" tabindex="10" aria-labelledby="#cookies">
```

Comment:
This removes the opt-out which @aliceh75  originally put in. Which is @willowo's preference (provide an opt-out, or don't provide an opt-out)?

If you remove this, you also need to remove the code which references it in trademapper/js/trademapper.analytics.js.